types.go

```diff
@@ -55,6 +55,11 @@ type (
 		// Checksum ensures the integrity of packages
 		// refereced by URL. Ignored for literals.
 		Checksum Checksum `json:"checksum"`
+	}
+
+	PackageRef struct {
+		PackageName      string
+		PackageNamespace string
```

Comment:
How does that Namespace+Name identification work with multiple versions of the same Package? Does the name get suffixed with a version?